### PR TITLE
Alternarive for bpo-29553 - Fixed ArgumentParses format_usage for mutually exclusive groups

### PR DESCRIPTION
**Pull request by andrewnester**
Fix for https://bugs.python.org/issue29553
Alternative to https://github.com/python/cpython/pull/117 
This alternative advised by Christoph Stahl and Brian Curtin

### CODE REVIEW

**Review by berkerpeksag — CHANGES_REQUESTED** [review]
See https://github.com/python/cpython/pull/117#discussion_r101918576 about tests I left in #117.

**Comment by autumnsashes3** [comment]
No mother f***** is only reason I'm on here is because you m************
keep goddamn f****** with people in real life and in just like in real life
really get somebody killed real f****** quick

**Comment by autumnsashes3** [comment]
I don't give two f**** what you guys have to do but fix it and I don't
f****** I hear another thing out of anybody ever the f*** again I swear to
God my niece and nephew get scared because they think somebody standing
outside doesn't matter what it is

**Comment by autumnsashes3** [comment]
I mean what kind of f****** person puts human s*** and f****** nasty trash
f****** nasty garbage smelling s*** in somebody's vehicle somebody that
obviously wants to be tortured very long and very f****** painfully

On Feb 19, 2017 6:21 AM, "Berker Peksag" <notifications@github.com> wrote:

*@berkerpeksag* requested changes on this pull request.

See #117 (comment)
<https://github.com/python/cpython/pull/117#discussion_r101918576> about
tests I left in #117 <https://github.com/python/cpython/pull/117>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/python/cpython/pull/120#pullrequestreview-22660718>,
or mute
the thread

**Comment by andrewnester** [comment]
@berkerpeksag thanks! just updated my PR.

**Comment by andrewnester** [comment]
@berkerpeksag any updates? :)

File: Misc/NEWS [hunk 1]
```diff
@@ -251,6 +251,7 @@ Library
 
 - bpo-29576: Improve some deprecations in importlib. Some deprecated methods
   now emit DeprecationWarnings and have better descriptive messages.
```
Comment (zhangyangyu) [inline Misc/NEWS]: We need an empty line here.

Comment (andrewnester) [reply Misc/NEWS]: @zhangyangyu updated, sorry! thanks!

**Comment by andrewnester** [comment]
@berkerpeksag any updates on this one? :)

**Comment by andrewnester** [comment]
@zhangyangyu @berkerpeksag any updates on this? :)

**Comment by brettcannon** [comment]
To try and help move older pull requests forward, we are going through and backfilling 'awaiting' labels on pull requests that are lacking the label. Based on the current reviews, the best we can tell in an automated fashion is that a core developer requested changes to be made to this pull request.

If/when the requested changes have been made, please leave a comment that says, `I have made the requested changes; please review again`. That will trigger a bot to flag this pull request as ready for a follow-up review.

**Comment by berkerpeksag** [comment]
Superseded by PR #14976.

**Pull request closed without merging**